packages/iris-grid/src/sidebar/visibility-ordering-builder/sortable-tree/SortableTreeItem.tsx

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable import/prefer-default-export */
+import React, { CSSProperties } from 'react';
+import type { UniqueIdentifier } from '@dnd-kit/core';
+import { AnimateLayoutChanges, useSortable } from '@dnd-kit/sortable';
+import { CSS } from '@dnd-kit/utilities';
+import { TreeItem, Props as TreeItemProps } from './TreeItem';
+import { iOS } from './utilities';
+
+export interface Props extends TreeItemProps {
+  id: UniqueIdentifier;
+}
+
+const animateLayoutChanges: AnimateLayoutChanges = ({
```

Comment:
Declare as `function`?

Reply:
Same as before. Not as clean in TS to declare as `function` since `AnimateLayoutChanges` is a function type